### PR TITLE
Explicit loading test

### PR DESCRIPTION
**Pull request by caleblloyd**
- Here is a test to display how Explicit Loading works
- It works out of the box with EF 1.1.0-preview1, no additional changes needed by us

### CODE REVIEW

File: test/Pomelo.EntityFrameworkCore.MySql.PerfTests/Tests/Models/CrmTest.cs [hunk 1]
```diff
@@ -79,5 +79,36 @@ public class CrmTest : IClassFixture<CrmFixture>
...
+		{
+			using (var db = new AppDb())
+			{
+				// load just the superUser
+				var superUser = await db.CrmAdmins.OrderByDescending(m => m.Id).FirstOrDefaultAsync();
+
+				// explicit load AdminMenus
+				await db.Entry(superUser).Collection(m => m.AdminMenus).LoadAsync();
+				// explicit load each Menu
+				foreach (var adminMenu in superUser.AdminMenus)
```
Comment (Kagamine) [inline test/Pomelo.EntityFrameworkCore.MySql.PerfTests/Tests/Models/CrmTest.cs]: Could you use Parallels.ForEach instead of foreach & await?

Comment (caleblloyd) [inline test/Pomelo.EntityFrameworkCore.MySql.PerfTests/Tests/Models/CrmTest.cs]: Oh yes, I think Task.WhenAll may be best. I will change that

Comment (Kagamine) [inline test/Pomelo.EntityFrameworkCore.MySql.PerfTests/Tests/Models/CrmTest.cs]: Thanks

**Comment by caleblloyd** [comment]
I think Microsoft may be bumping from 1.1.0-preview1 to 1.1.0 soon, which is why AppVeyor is failing.  I will wait until I can fix build issues to merge.

**Pull request merged**